[2024-11-02T06:26:57.486+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T06:26:57.537+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-11-02T06:26:57.567+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-11-02T06:26:57.570+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-02T06:26:57.635+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-11-01 00:00:00+00:00
[2024-11-02T06:26:57.682+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=195) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-02T06:26:57.692+0000] {standard_task_runner.py:72} INFO - Started process 197 to run task
[2024-11-02T06:26:57.690+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'scheduled__2024-11-01T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpoaxzll9s']
[2024-11-02T06:26:57.697+0000] {standard_task_runner.py:105} INFO - Job 118: Subtask second_task
[2024-11-02T06:26:57.838+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.second_task scheduled__2024-11-01T00:00:00+00:00 [running]> on host 2d730dc1e71d
[2024-11-02T06:26:58.192+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-01T00:00:00+00:00'
[2024-11-02T06:26:58.193+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T06:26:58.242+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-02T06:26:58.243+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am the second task after task 1']
[2024-11-02T06:26:58.272+0000] {subprocess.py:86} INFO - Output:
[2024-11-02T06:26:58.285+0000] {subprocess.py:93} INFO - hey, I am the second task after task 1
[2024-11-02T06:26:58.286+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-02T06:26:58.371+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T06:26:58.374+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, run_id=scheduled__2024-11-01T00:00:00+00:00, execution_date=20241101T000000, start_date=20241102T062657, end_date=20241102T062658
[2024-11-02T06:26:58.496+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T06:26:58.524+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
